0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -98,47 +100,46 @@ Ledger protocols are responsible for executing the individual transfers that con
+The ledger MUST provide cryptographic authorization holds. During an authorization hold, money is put aside for a specific transfer until that transfer's outcome has been decided.
+
+Transfers using authorization holds can be in four distinct states:
 
-![enter image description here](https://lh3.googleusercontent.com/-QHsehrPTXgM/Vu_zs1dLWPI/AAAAAAAAEys/zchHKJj_MQs/s0/Transfer%252520States.png "Transfer States")
+![Transfers start in the proposed state, transition to the prepared state and finally to the executed state, which is final. Proposed and prepared transfers can also transition to the rejected state, which is final as well.](../shared/graphs/transfer-states.svg)
 
 * Proposed -- Nothing has happened yet.
-* Prepared -- Funds are held in escrow.
+* Prepared -- Funds are held.
 * Executed -- The transfer has completed.
 * Rejected -- The transfer has been canceled (and funds returned to the sender.)
 
-<span class="show alert alert-info">**Hint:** Escrow is the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
+<span class="show alert alert-info">**Hint:** Authorization holds are the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
 
-
-The ledger MUST be able to verify a simple SHA-256 hashlock cryptographic escrow condition in order to automatically trigger the release of the escrow. It must also allow a timeout to automatically roll back an escrowed transfer.
+The ledger MUST be able to release the held funds to the receiver upon receiving a 32-byte preimage whose SHA-256 hash matches a value provided by the sender. It MUST also support releasing the funds back to the sender after a timeout.
 
 The ledger MUST support attaching a short message or *memo* to each transfer.
 
 #### Full Support
 
 The ledger MUST fulfill the requirements for basic support and also the following:
 
-It MUST support all cryptographic condition types with the status "recommended".
-
-It MUST support memos to be added for the credited and debited accounts separately and SHOULD support fairly large memo sizes.
+It MUST support memos up to 65535 bytes.
 
-It MUST support a way to discover connectors connected to the ledger.
+It MUST support sending an authenticated message of up to 65535 bytes to the holder of another account on the ledger.
 
-It MUST support a way to look up a fulfillment by condition hash. It SHOULD automatically reject new transfers (that have not been prepared yet) that have an execution condition for which the ledger already knows the fulfillment. This aids in [UTP error recovery](#error-recovery).
+It MUST support a way to look up a fulfillment by condition hash. It SHOULD automatically reject new transfers (that have not been prepared yet) that have an execution condition for which the ledger already knows the fulfillment. This aids in [error recovery](#error-recovery).
 
```

Comment:
We should mention that it is necessary for a ledger protocol to define an addressing scheme that complies with the ILP Addressing scheme such that accounts in the ledger can be addressed using ILP Addresses.

See: https://github.com/interledger/rfcs/issues/31#issuecomment-286469564